app/models/proxy.rb

```diff
@@ -559,12 +560,16 @@ def proxy
   class PolicyConfig
     include ActiveModel::Validations
 
+    attr_reader :init_attributes
     attr_accessor :name, :version, :configuration, :enabled
-    validates :name, :version, presence: true
-    validate :configuration_is_object
+
+    validates :name, :version, presence: true, if: :attributes_are_hash?
```

Comment:
Why not raise an error if passed a wrong argument?

Reply:
QE is complaining about 500.. so that wouldn't solve the problem.